client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -40,6 +40,11 @@ const availableFlows: Record< string, () => Promise< { default: Flow } > > = {
 			/* webpackChunkName: "tailored-ecommerce-flow" */ '../declarative-flow/tailored-ecommerce-flow'
 		),
 
+	ecommerceTrial: () =>
```

Comment:
I think we should call this something more like `wooExpress`, as we're really building a WooExpress-specific signup flow.